### PR TITLE
[Feature]Temp Ban those who keep quitting

### PR DESCRIPTION
**Pull request by NikoCat233**
Temp ban those who keep quitting
Directly temp ban low level players
This system uses while list
adding translations to crowdin later =)

Maybe tempbanwhitelist will be useful

### CODE REVIEW

**Comment by PHGaming121** [comment]
really good idea

File: Modules/OptionHolder.cs [hunk 1]
```diff
@@ -2670,6 +2677,11 @@ public static void Load()
             .SetValueFormat(OptionFormat.Times);
         AutoKickStartAsBan = BooleanOptionItem.Create(19312, "AutoKickStartAsBan", false, TabGroup.SystemSettings, false)
             .SetParent(AutoKickStart);
+
+        TempBanPlayersWhoKeepQuitting = BooleanOptionItem.Create(44440, "TempBanPlayersWhoKeepQuitting", false, TabGroup.SystemSettings, false);
+        QuitTimesTillTempBan = IntegerOptionItem.Create(44441, "QuitTimesTillTempBan", new(1, 15, 1), 4, TabGroup.SystemSettings, false)
+            .SetParent(TempBanPlayersWhoKeepQuitting)
+            .SetValueFormat(OptionFormat.Times);
```
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:2684]: This setting is not used anywhere, lol

File: Modules/OptionHolder.cs [hunk 2]
```diff
@@ -2670,6 +2677,11 @@ public static void Load()
             .SetValueFormat(OptionFormat.Times);
         AutoKickStartAsBan = BooleanOptionItem.Create(19312, "AutoKickStartAsBan", false, TabGroup.SystemSettings, false)
             .SetParent(AutoKickStart);
+
+        TempBanPlayersWhoKeepQuitting = BooleanOptionItem.Create(44440, "TempBanPlayersWhoKeepQuitting", false, TabGroup.SystemSettings, false);
+        QuitTimesTillTempBan = IntegerOptionItem.Create(44441, "QuitTimesTillTempBan", new(1, 15, 1), 4, TabGroup.SystemSettings, false)
+            .SetParent(TempBanPlayersWhoKeepQuitting)
```
Comment (Tommy-XL) [inline Modules/OptionHolder.cs:2683]: And it's unnecessary

File: Modules/OptionHolder.cs (hunk 1 above)
Comment (NikoCat233) [reply Modules/OptionHolder.cs:2684]: Huh?
(Main.PlayerQuitTimes[data?.FriendCode] >= Options.QuitTimesTillTempBan.GetInt())

File: Modules/OptionHolder.cs (hunk 2 above)
Comment (NikoCat233) [reply Modules/OptionHolder.cs:2683]: Yeah delete it

File: Modules/OptionHolder.cs (hunk 1 above)
Comment (Tommy-XL) [reply Modules/OptionHolder.cs:2684]: Damn, there are 2 settings and not 1, lol

But `TempBanPlayersWhoKeepQuitting` not using

File: Modules/OptionHolder.cs (hunk 2 above)
Comment (Tommy-XL) [reply Modules/OptionHolder.cs:2683]: My bad, I thought there was only 1 setting

File: Modules/OptionHolder.cs (hunk 1 above)
Comment (Tommy-XL) [reply Modules/OptionHolder.cs:2684]: Or wait, if `TempBanPlayersWhoKeepQuitting` is false, `QuitTimesTillTempBan` will be work?

**Comment by Tommy-XL** [comment]
I think this will be merged after #133  is merged, because additional bugs may be found

**Review by 0xDrMoe — APPROVED** [review]

**Pull request merged**